server/mailet/mailetcontainer-camel/src/main/java/org/apache/james/mailetcontainer/impl/LocalResources.java

```diff
@@ -58,8 +58,7 @@ boolean isLocalServer(Domain domain) {
         try {
             return domains.containsDomain(domain);
         } catch (DomainListException e) {
-            LOGGER.error("Unable to retrieve domains", e);
-            return false;
+            throw new RuntimeException(e);
```

Comment:
Can't we keep the message as the first parameter of the RuntimeException constructor ?